### PR TITLE
Update "Flux in Short" section

### PR DESCRIPTION
**Pull request by dholbach**
- link to OCI resources
- mention Ecosystem additions: integrations, cloud offerings, UIs

Fixes: #1157

Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

**Comment by dholbach** [comment]
I reviewed #1157 and found that there's better places to link to the e2e doc from, so the above should be it for now.

If somebody gives me a :+1: for this, I'll update `content/en/_index.html` and `templates/monthly-update.md` as well.

**Comment by dholbach** [comment]
→ https://deploy-preview-1376--fluxcd.netlify.app/flux/#flux-in-short

**Comment by dholbach** [comment]
This should be ready for review now.

→ https://deploy-preview-1376--fluxcd.netlify.app/

**Comment by kingdonb** [comment]
Looks like there is a minor change needed:
Screenshot 2023-02-15 at 9 20 02 AM

the punctuation is inconsistent (sentences are all ended with a period here) and the pipe character shows up where it ought not be visible, I think this is one of the pipes used for table formatting.

File: content/en/_index.html [hunk 1]
```diff
@@ -61,6 +61,10 @@ <h2 id="flux-facts">Flux in short</h2>
     Flux uses true Kubernetes RBAC via impersonation and supports multiple Git repositories. Multi-cluster infrastructure and apps work out of the box with Cluster API: Flux can use one Kubernetes cluster to manage apps in either the same or other clusters, spin up additional clusters themselves, and manage clusters including lifecycle and fleets.
   {{< /blocks/featureitem >}}
 
+  {{< blocks/featureitem emoji="✨" caption="Dashboards love Flux" >}}
+    No matter if you use one of <a href="/ecosystem/#flux-uis--guis">the Flux UIs</a> or a hosted cloud offering from your cloud vendor, Flux has a thriving ecosystem of integrations and products built on top of it and all have great dashboards for you |
```
Comment (kingdonb) [inline content/en/_index.html:65]:
```suggestion
    No matter if you use one of <a href="/ecosystem/#flux-uis--guis">the Flux UIs</a> or a hosted cloud offering from your cloud vendor, Flux has a thriving ecosystem of integrations and products built on top of it and all have great dashboards for you.
```

File: content/en/flux/_index.md [hunk 2]
```diff
@@ -67,10 +67,11 @@ If you want to find out more about Flux UIs, check out our [dedicated section](/
...
 | 🔒 Flux is designed with security in mind | Pull vs. Push, least amount of privileges, adherence to Kubernetes security policies and tight integration with security tools and best-practices. Read more about [our security considerations](/flux/security). |
 | ☸️ Flux works with any Kubernetes and all common Kubernetes tooling |  Kustomize, Helm, RBAC, and policy-driven validation (OPA, Kyverno, admission controllers) so it simply falls into place. |
 | 🤹 Flux does Multi-Tenancy (and “Multi-everything”) | Flux uses true Kubernetes RBAC via impersonation and supports multiple Git repositories. Multi-cluster infrastructure and apps work out of the box with Cluster API: Flux can use one Kubernetes cluster to manage apps in either the same or other clusters, spin up additional clusters themselves, and manage clusters including lifecycle and fleets. |
+| ✨ Dashboards love Flux | No matter if you use one of [the Flux UIs](/ecosystem/#flux-uis--guis) or a hosted cloud offering from your cloud vendor, Flux has a thriving ecosystem of integrations and products built on top of it and all have great dashboards for you |
```
Comment (kingdonb) [inline content/en/flux/_index.md:74]:
```suggestion
| ✨ Dashboards love Flux | No matter if you use one of [the Flux UIs](/ecosystem/#flux-uis--guis) or a hosted cloud offering from your cloud vendor, Flux has a thriving ecosystem of integrations and products built on top of it and all have great dashboards for you. |
```

**Review by kingdonb — APPROVED** [review]
I think that does it 👐

**Comment by dholbach** [comment]
Nice thanks for the edits and your attention to detail!

**Pull request merged**